fn_falcon_sandbox/fn_falcon_sandbox/util/submit_helper.py

```diff
@@ -0,0 +1,117 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+import time
+import requests
+import tempfile
+import os
+from requests_toolbelt import MultipartEncoder
+from resilient_circuits import FunctionError
+from fn_falcon_sandbox.util.constants import (
+    HA_AVAILABLE_ENVS,
+    HA_AVAILABLE_RUNTIME_ACTION_SCRIPTS,
+)
+
+
+def write_temp_file(data, name=None):
+    """This function writes content to a temporary file and 
+        returns file path and list of files.
+    """
+    temp_files = []
+    path = None
+
+    if name:
+        path = "{0}/{1}".format(tempfile.gettempdir(), name)
```

Comment:
* This will cause issues when ran on Windows
* Use https://docs.python.org/2.7/library/os.path.html?highlight=os%20path%20join#os.path.join